.github/workflows/build_llvm_libraries.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: Reusable workflow-build_llvm_libraries&build_lldb_if_needed
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        required: true
+        # can't pass array now, workaround by passing Json string and deserialize later
+        default: "['ubuntu-20.04', 'ubuntu-22.04']"
```

Comment:
if `required` is `true`, `default` values will never be used. may consider to remove `default`.